src/main/resources/db/migration/data/V1__add_protocols_table.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE protocols (
     id CHARACTER VARYING(255) NOT NULL,
-    version INTEGER NOT NULL,
+    version VARCHAR(20) NOT NULL CHECK (version ~ '^(\d+\.)?(\d+\.)?(\*|\d+)$'),
```

Comment:
I did this here because of the tests, but I guess the correct way would be to create a migration?